packages/docs/src/pages/startup/installation.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Installation
+weight: 0
+---
+
+<h2 id="installation" class="ds-h2 ds-u-color--primary-darker">Installation</h2>
+
+The design system is available as NPM packages or via a <a href="https://github.com/CMSgov/design-system/archive/v1.30.0.zip">.zip download</a>.
+
+The design system consists of two packages which are installed separately.
+
+The [**core** package](https://www.npmjs.com/package/@cmsgov/design-system-core) includes the bulk of the design system:
+
+* Base styles
+* Utility classes
+* Sass/CSS and React components
+* Sass mixins and variables
+* Fonts and images
+
+```
+npm install --save @cmsgov/design-system-core
+```
+
+The [**layout** package](https://www.npmjs.com/package/@cmsgov/design-system-layout) includes:
+
+* Responsive flexbox grid framework
+
+```
+npm install --save @cmsgov/design-system-layout
+```
+
+<h2 id="need-help" class="ds-h2 ds-u-color--primary-darker">Need more information on NPM?</h2>
+<p class="ds-c-alert__text">Learn more about using packages and NPM at <a href="https://www.npmjs.com/">npmjs.org</a>.</p>
+
+<h3>Usage</h3>
```

Comment:
I think this should be an h2 (it seems like it's the same level as the "Need more information on NPM" h2 above it)

Reply:
Agree. Bumped it up and bumped the NPM header down one. Interestingly, Markdown's not rendering, so leaving it as full html markup.